action_tutorials/action_tutorials_cpp/README.md

```diff
@@ -0,0 +1,67 @@
+# Action Server
+
+In the constructor for `FibonacciActionServer`, an action server is created with callbacks that are called when a goal is received, when the goal is cancelled, and when the goal is accepted:
+
+```cpp
+this->action_server_ = rclcpp_action::create_server<Fibonacci>(
+  ...
+  "fibonacci",
+  std::bind(&FibonacciActionServer::handle_goal, this, _1, _2),
+  std::bind(&FibonacciActionServer::handle_cancel, this, _1),
+  std::bind(&FibonacciActionServer::handle_accepted, this, _1));
+```
+
+The `handle_goal` callback is called whenever a goal is sent to the action server by an action client.
+In the example code, the goal is accepted as long as the order is less than 46, otherwise it is rejected.
+This is to prevent potential [integer overflow](https://en.wikipedia.org/wiki/Integer_overflow):
+```cpp
+if (goal->order > 46) {
+  return rclcpp_action::GoalResponse::REJECT;
+}
+return rclcpp_action::GoalResponse::ACCEPT_AND_EXECUTE;
+```
+
+The `handle_cancelled` callback is called whenever an action client requests to cancel the goal being executed.
+In this case, the goal cancel request is always accepted.
+
+The `handle_accepted` callback is called following the action server's acceptance of a goal. In this example, a thread is spun-up to execute the goal in the background:
```

Comment:
A couple suggestions here:
* For the sake of non-native English speakers, I might avoid the term "spun-up". Instead it could be "started" or "launched." 
* I'd also avoid saying "in the background" as this might imply a lower (background) priority for the thread.